CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -10,6 +10,10 @@ Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/
 
 ## v3.6.0-rc.3 (TBD)
 
+### Package `clientv2`
```

Comment:
are we still even planning to cut new v2 tags for the client?

right now, the last non-prerelease tag in the v2 stream (https://github.com/etcd-io/etcd/tree/client/v2.305.0/client/v2) is a usable module ... I would sort of expect to just leave that alone and not even cut a client/v2.306.0 tag at all

Reply:
> are we still even planning to cut new v2 tags for the client?

No, we won't create new tags for client/v2, because it's gone.

Reply:
Updated the changelog & thx